scripts/llext_link_helper.py

```diff
@@ -76,6 +76,7 @@ def main():
 	executable = []
 	writable = []
 	readonly = []
+	readonly_dram = []
```

Comment:
I assume this will all be dumped when we build the target i.e. xkB hybrid RODATA in DRAM ?

Reply:
@lgirdwood this isn't implemented yet, should be doable incrementally, yes